modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -10,6 +10,15 @@ import {xSmallIcon} from '@workday/canvas-system-icons-web';
 import {TextInputProps} from '@workday/canvas-kit-react-text-input';
 import uuid from 'uuid/v4';
 
+export interface ComboBoxMenuItemProps extends MenuItemProps {
+  shortText?: string;
+}
```

Comment:
Not 100% sold on how I think the api for this should be. Should have to extend a menu item. Which feels kinda awkward. Or maybe combobox should optionally read from menuItem attribute, e.g. title.